pybop/problems/base_problem.py

```diff
@@ -120,7 +121,7 @@ def observed_fisher(self, x: np.ndarray) -> np.ndarray:
         """
         raise NotImplementedError
 
-    def run(self) -> float:
+    def run(self) -> np.ndarray:
```

Comment:
its unclear why run is returning an np.ndarray rather than a float, need to update docstring and specify what the length of the array is. Also, shouldn't `run_with_sensitivities` return a np.ndarray rather than a float as well? You'll need to update this docstring as well to specify the dimensions of each return value of `run_with_sensitivities`.